src/dashboard/src/components/api/views.py

```diff
@@ -500,6 +475,8 @@ def reingest(request, target):
         response = {'message': 'Approval successful.', 'reingest_uuid': reingest_uuid}
         return helpers.json_response(response)
 
+# TODO should this have auth?  Does it support GET?
+@_api_endpoint(expected_methods=['POST'])
 def copy_metadata_files_api(request):
```

Comment:
This had no auth or verb restriction previously - needs to be tested to see if it still works with external consumers.